### ISSUE TITLE
No line segments in plot causes QPainter error

### ISSUE THREAD

**Issue by Obliman** [pyqtgraph/pyqtgraph#2541]
### Short description

When creating a simple plot with only one point `i.e. plot([0], [0], pen=mkPen('r', width=2))` I encounter an error stating that since there are no line segments present the QPainter is missing its required arg(s). This only seems to happen when a pen is also defined for the plot. This error arises during runtime whenever exactly one point is present on the plot.

### Code to reproduce

```
from PySide6.QtWidgets import *
from pyqtgraph import mkPen, PlotWidget


class MainWindow(QMainWindow):
    def __init__(self):
        super().__init__()

        self.layout = QVBoxLayout()

        self.fig = PlotWidget()
        self.fig.plot([0], [0], pen=mkPen("r", width=2))

        self.layout.addWidget(self.fig)

        self.main_widget = QWidget()
        self.main_widget.setLayout(self.layout)
        self.setCentralWidget(self.main_widget)


if __name__ == "__main__":
    app = QApplication()

    window = MainWindow()
    window.show()

    app.exec()
```

### Expected behavior
No error, just plotting

### Real behavior
An error because no line segments exist with only one point.

```
/home/user/.local/lib/python3.10/site-packages/pyqtgraph/debug.py:128: RuntimeWarning: Ignored exception:
Traceback (most recent call last):
  File "/home/user/path/testing2.py", line 277, in <module>
    app.exec()
  File "/home/user/.local/lib/python3.10/site-packages/pyqtgraph/widgets/GraphicsView.py", line 137, in paintEvent
    return super().paintEvent(ev)
  File "/home/user/.local/lib/python3.10/site-packages/pyqtgraph/debug.py", line 128, in w
    printExc('Ignored exception:')
  --- exception caught here ---
  File "/home/user/.local/lib/python3.10/site-packages/pyqtgraph/debug.py", line 126, in w
    func(*args, **kwds)
  File "/home/user/.local/lib/python3.10/site-packages/pyqtgraph/graphicsItems/PlotCurveItem.py", line 905, in paint
    p.drawLines(*self._getLineSegments())
TypeError: QPainter.drawLines() takes exactly one argument (0 given)
  printExc('Ignored exception:')
```

### Tested environment(s)

 * PyQtGraph version: 0.13.1
 * Qt Python binding: PySide6 6.4.0.1 Qt 6.4.0
 * Python version: 3.10.7
 * NumPy version: 1.21.5
 * Operating system: Linux Mint 21 (Ubuntu 22.04)
 * Installation method: pip

**Comment by pijyoi** [comment]
This should have been fixed in #2461

**Comment by Obliman** [comment]
> This should have been fixed in 2481
I made the change described there and can confirm that it addressed the issue. It just hasn't been included in a pypi release yet.
